src/Resources/public/sonata-page.back.js

```diff
@@ -491,6 +491,9 @@
                     url:     url,
                     data:    $form.serialize(),
                     type:    method,
+                    headers: {
+                        Accept: 'text/html, application/xhtml+xml;'
+                    },
                     success: function (resp) {
                         $loader.hide();
                         if (resp.result && resp.result === 'ok') {
```

Comment:
If the accepted type is only HTML, how is this condition supposed to work?

Reply:
Honestly, I know nothing at all about this.

The original issue was 
https://github.com/sonata-project/SonataPageBundle/issues/1318
a similar is now for edit
https://github.com/sonata-project/SonataPageBundle/issues/1376

I just did the PR as asked by @gremo

Reply:
@phansys @VincentLanglet the explanation of the fix was in the original issue https://github.com/sonata-project/SonataPageBundle/issues/1318#issuecomment-855810425 if you have some question just ask